launcher/modManager/chroniclesextractor.cpp

```diff
@@ -147,14 +133,13 @@ void ChroniclesExtractor::createChronicleMod(int no)
 	dir.removeRecursively();
 	dir.mkpath(".");
 
-	QByteArray tmpChronicles = chronicles.at(no);
-	tmpChronicles.replace('\0', "");
+	QString tmpChronicles = chronicles.at(no);
 
 	QJsonObject mod
 	{
 		{ "modType", "Expansion" },
-		{ "name", QString::number(no) + " - " + QString(tmpChronicles) },
-		{ "description", tr("Heroes Chronicles") + " - " + QString::number(no) + " - " + QString(tmpChronicles) },
+		{ "name", QString::number(no) + " - " + tmpChronicles },
+		{ "description", tr("Heroes Chronicles") + " - " + QString::number(no) + " - " + tmpChronicles },
```

Comment:
Please use QString{}.arg()

Reply:
Wanted to avoid localization changes... I guess I can also adjust .ts files manually.

Reply:
It's also an incorrect way to write localized strings (didn't notice them first). The right way is tr("something %1 etc", "comment about parameters might also be useful").arg()